From: Shengjing Zhu <i@zhsj.me>
Date: Sat, 15 Apr 2017 10:52:58 +0000
Subject: disable tests

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0c7394e..c5c86dc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,6 +40,6 @@ include_directories(
 add_subdirectory(doc)
 add_subdirectory(data)
 add_subdirectory(include)
-add_subdirectory(tests)
+#add_subdirectory(tests)
 
 # enable_coverage_report(posix_process_test linux_process_test)
